pyasdf/tests/test_utils.py

```diff
@@ -93,3 +93,8 @@ def test_sizeof_fmt_function():
     assert sizeof_fmt(1024 ** 2) == "1.0 MB"
     assert sizeof_fmt(1024 ** 3) == "1.0 GB"
     assert sizeof_fmt(1024 ** 4) == "1.0 TB"
+
+
+def test_replace_None():
+    a = replace_None({u'user': None})
+    assert a[u'user']
```

Comment:
This assertion could be a bit more specific, e.g. `assert X is None` or `assert isinstance(X, h5py.Empty)`.